pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -87,7 +89,8 @@ type LoadBalancerOpts struct {
 }
 
 type BlockStorageOpts struct {
-	TrustDevicePath bool `gcfg:"trust-device-path"` // See Issue #33128
+	BSVersion       string `gcfg:"bs-version"`        // overrides autodetection. v1 or v2. Defaults to auto
```

Comment:
This was done to match already existing code. See loadbalancer's LoadBalancerOpts